Assets/MRTK/Core/Providers/BaseSpatialMeshObserver.cs

```diff
@@ -281,6 +282,8 @@ public Material OcclusionMaterial
             }
         }
 
+        public PhysicMaterial PhysicsMaterial  { get; private set; }
```

Comment:
Applications generally do not access these objects via the base or concrete classes. This property should be on an interface. That said, it would be a breaking change to modify the IMixedRealitySpatialAwarenessMeshObserver interface.

To address this, please consider making a new interface. Something descriptive, such as ISpatialAwarenessPhysicsProperties. Applications would cast to this interface to set/get these values in the same manner as is done for the capability check functionality.